shopfloor_reception/services/reception.py

```diff
@@ -0,0 +1,1079 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+
+from odoo import fields
+
+from odoo.addons.base_rest.components.service import to_int
+from odoo.addons.component.core import Component
+from odoo.addons.shopfloor.utils import to_float
+
+
+class Reception(Component):
+    """
+    Methods for the Reception Process
+
+    You can find a sequence diagram describing states and endpoints relationships
+    [here](../docs/reception_sequence_graph.png)
+    Keep [the sequence diagram](../docs/reception_sequence_graph.mermaid)
+    up-to-date if you change endpoints.
+
+    Process a picking and track progress by product.
```

Comment:
Better to fit with what we are processing with this scenario, and re-use Odoo terms:
```suggestion
    Process a receipt transfer and track progress by product.
```

Reply:
Updated.
I'm assuming this scenario description is the only place we should update, as this is referred to as "picking" in the rest of the module right?

Reply:
Yes, only in the docstring.
I think the right fix would be to rename `stock.picking` to `stock.transfer`, as picking is a type of transfer, but we won't win this battle I guess :)